src/view/utils/getDefaultIdentity.js

```diff
@@ -0,0 +1,7 @@
+export default () => {
```

Comment:
Add Copyright notice. Rename this to getDefaultIdentifier

Reply:
thank you